index.html

```diff
@@ -1177,119 +1177,269 @@ <h2 id="relationship-to-permissions-policy">
-          <li>Wait for all <a>tasks</a> in |tasks| to have executed.
-          </li>
-          <li>Return <a>success</a> with data `null`.
-          </li>
-        </ol>
-        <aside class="example" title="Setting a permission via WebDriver">
+        <section>
+          <h4 id="extension-module-permissions">
+            The permissions Module
+          </h4>
           <p>
-            To [=extension commands/set permission=] for `{name: "midi", sysex: true}` of the
-            [=current settings object=] of the <a>session</a> with ID 23 to "`granted`", the local
-            end would POST to `/session/23/permissions` with the body:
+            The <dfn>permissions</dfn> module contains commands for
+            managing the remote end browser permissions.
           </p>
-          <pre class="lang-json">
-          {
-            "descriptor": {
-              "name": "midi",
-              "sysex": true
-            },
-            "state": "granted"
-          }
-          </pre>
-        </aside>
+          <section>
+            <h5 id="extension-module-permissions-definition">
+              Definition
+            </h5>
+            <p>
+              [=remote end definition=]
+            </p>
+            <pre class="cddl remote-cddl">
+              PermissionsCommand = (
+                permissions.setPermission
+              )
+            </pre>
+          </section>
+          <section>
+            <h5 id="extension-module-permissions-types">
+              Types
+            </h5>
+            <section>
+              <h6 id="extension-module-type-permissions-PermissionDescriptor">
+                The permissions.PermissionDescriptor Type
+              </h6>
+              <pre class="cddl local-cddl">
+                permissions.PermissionDescriptor = {
+                  name: text,
+                }
+              </pre>
+              <p>
+                The <code>permissions.PermissionDescriptor</code> type represents a {{PermissionDescriptor}}.
+              </p>
+            </section>
+            <section>
+              <h6 id="extension-module-type-permissions-PermissionState">
+                The permissions.PermissionState Type
+              </h6>
+              <pre class="cddl local-cddl">
+                permissions.PermissionState = "granted" / "denied" / "prompt"
+              </pre>
+              <p>
+                The <code>permissions.PermissionState</code> type represents a {{PermissionState}}.
+              </p>
+            </section>
+          </section>
+          <section>
+            <h5 id="extension-module-permissions-commands">
+              Commands
+            </h5>
+            <section>
+              <h6 id="extension-module-command-permissions-setPermission">
+                The permissions.setPermission Command
+              </h6>
+              <p>
+                The <dfn class="export" data-dfn-for="extension modules">Set Permission</dfn>
```

Comment:
I think we should remove the link to the extension module definition. You can link to https://w3c.github.io/webdriver-bidi/#module instead

Reply:
Done